janrg/badiDate
# Coverage

- Refactoring: Switch to `moment` global
- Refactoring: Remove unneeded "type=text/javascript"
- Linting (ESLint): Disable newly applied "standard" rule to avoid breakage (`object-curly-newline`); add babel-eslint for dynamic `import()`
- Testing: Add coverage
- Testing: Switch from node-qunit to mocha
- Build: Avoid stripping imports which are no longer present
- Build: Suppress some rollup warnings by specifying "moment-timezone" as external
- npm: Add browser testing script
- npm: Update devDeps and meeussunmoon and moment-timezone deps.
